sdk/python/v1beta1/kubeflow/katib/api/katib_report_metrics.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2024 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from datetime import datetime
+from typing import Any, Dict
+
+import grpc
+import pytz
```

Comment:
Is it possible to not use `pytz` for date-time format ? E.g. we can add the format to the SDK constants.
That will help us to reduce number of dependencies for Katib SDK.

Reply:
Yes, I will import `timezone` from `datetime` package to accomplish that.